rotkehlchen/constants/assets.py

```diff
@@ -275,10 +275,40 @@
 A_SAND = Asset('eip155:1/erc20:0x3845badAde8e6dFF049820680d1F14bD3903a5d0')
 A_GLM = Asset('eip155:1/erc20:0x7DD9c5Cba05E151C895FDe1CF355C9A1D5DA6429')
 
+# hop protocol
+A_MAGIC = Asset('eip155:1/erc20:0xB0c7a3Ba49C7a6EaBa6cD4a96C55a1391070Ac9A')
+A_RETH = Asset('eip155:1/erc20:0xae78736Cd615f374D3085123A210448E74Fc6393')
+A_HOP = Asset('eip155:1/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
+A_USDC_ARB = Asset('eip155:42161/erc20:0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8')
+A_USDT_ARB = Asset('eip155:42161/erc20:0xFd086bC7CD5C481DCC9C85ebE478A1C0b69FCbb9')
+A_DAI_ARB = Asset('eip155:42161/erc20:0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1')
+A_HOP_ARB = Asset('eip155:42161/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
+A_RETH_ARB = Asset('eip155:42161/erc20:0xEC70Dcb4A1EFa46b8F2D97C310C9c4790ba5ffA8')
+A_MAGIC_ARB = Asset('eip155:42161/erc20:0x539bdE0d7Dbd336b79148AA742883198BBF60342')
+A_USDC_BASE = Asset('eip155:8453/erc20:0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA')
+A_HOP_BASE = Asset('eip155:8453/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
+A_USDC_GNOSIS = Asset('eip155:100/erc20:0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83')
+A_USDT_GNOSIS = Asset('eip155:100/erc20:0x4ECaBa5870353805a9F068101A40E0f32ed605C6')
+A_MATIC_GNOSIS = Asset('eip155:100/erc20:0x7122d7661c4564b7C6Cd4878B06766489a6028A2')
+A_HOP_GNOSIS = Asset('eip155:100/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
+A_USDC_OPT = Asset('eip155:10/erc20:0x7F5c764cBc14f9669B88837ca1490cCa17c31607')
+A_USDT_OPT = Asset('eip155:10/erc20:0x94b008aA00579c1307B0EF2c499aD98a8ce58e58')
+A_DAI_OPT = Asset('eip155:10/erc20:0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1')
+A_HOP_OPT = Asset('eip155:10/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
+A_RETH_OPT = Asset('eip155:10/erc20:0x9Bcef72be871e61ED4fBbc7630889beE758eb81D')
+A_SUSD_OPT = Asset('eip155:10/erc20:0x8c6f28f2F1A3C87F0f938b96d27520d9751ec8d9')
+A_SNX_OPT = Asset('eip155:10/erc20:0x8700dAec35aF8Ff88c16BdF0418774CB3D7599B4')
+A_USDC_POLYGON = Asset('eip155:137/erc20:0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174')
+A_USDT_POLYGON = Asset('eip155:137/erc20:0xc2132D05D31c914a87C6611C10748AEb04B58e8F')
+A_DAI_POLYGON = Asset('eip155:137/erc20:0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063')
+A_HOP_POLYGON = Asset('eip155:137/erc20:0xc5102fE9359FD9a28f877a67E36B0F050d81a3CC')
```

Comment:
Are all this assets already in the globaldb? Why do you need them to be here?

Reply:
I added them for ease of using them in the bridges, basically can know what Asset is used for which bridge.

Reply:
The problem of adding assets here is that they live in memory all the time and we do special checks for them. In your case I believe you only compare it for the identifier so move them to the constants files of the chains where they live